airbyte-cdk/python/airbyte_cdk/sources/declarative/extractors/response_to_file_extractor.py

```diff
@@ -15,7 +15,7 @@
 
 EMPTY_STR: str = ""
 DEFAULT_ENCODING: str = "utf-8"
-DOWNLOAD_CHUNK_SIZE: int = 1024 * 1024 * 10
+DOWNLOAD_CHUNK_SIZE: int = 1024 * 10
```

Comment:
[The current Salesforce code was using 1 Mb](https://github.com/airbytehq/airbyte/blob/4043fe072c9b5b89b351f584cd248634ae8aa324/airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py#L511) which seemed a bit low so I increase that to 10 Mb